cmd/parse/gov/proposal.go

```diff
@@ -78,7 +79,7 @@ func proposalCmd(parseConfig *parsecmdtypes.Config) *cobra.Command {
 				return fmt.Errorf("error while getting chain latest block height: %s", err)
 			}
 
-			err = govModule.UpdateProposal(height, proposalID)
+			err = govModule.UpdateProposal(height, time.Time{}, proposalID)
```

Comment:
Shouldn't this be `time.Now` instead of `time.Time`? Otherwise the check `passedVotingPeriod := blockTime.After(proposal.VotingEndTime)` will always be false considering that `time.Time{}` represents January 1st 1997